tests/conftest.py

```diff
@@ -313,14 +313,15 @@ def docker_image(boto_model_file):
         pass
 
     if boto_model_file is None:
-        raise ValueError('boto_model_file cannot be None')
+        print('boto_model_file is None, using default model.')
```

Comment:
Why we no longer want to raise error here?

Reply:
the purpose of the boto_model_file argument is to allow the docker image to use an unreleased version of the API, in most cases we just want to use the default. I added this raise during an earlier refactoring, initially the behavior assumed the argument was passed and failed if not given.